scripts/config_exadg.sh

```diff
@@ -37,6 +37,8 @@ FFTW=$WORKING_DIRECTORY/fftw/install
 LIKWID=$WORKING_DIRECTORY/likwid/install
 
 cmake \
+    -D CMAKE_BUILD_TYPE="DebugRelease" \
```

Comment:
What is the meaning of `DebugRelease` here? In the CMakeLists.txt file we have `ADD_CUSTOM_TARGET` where we can switch between debug and release?